### PR TITLE
Google app script

### PR DESCRIPTION
**Pull request by tatyree**
This has to be manually installed and configured on each form at the moment.  This PR is just here to centralise the code and the instructions.

### CODE REVIEW

**Comment by tatyree** [comment]
[As a JAC admin in charge of a test, I want to be able to plug in a Google App Script module/plugin that writes directly to firestore.](https://trello.com/c/YaHZ6LCe/9-as-a-jac-admin-in-charge-of-a-test-i-want-to-be-able-to-plug-in-a-google-app-script-module-plugin-that-writes-directly-to-firest)

File: google-app-scripts/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Google App Scripts for test support
+
+[Google App Scripts](https://developers.google.com/apps-script/) are cloud functions that are tightly tied to various G-Suite
+apps. At best these are diffcult to test and manage and nomrally have to be accesses and configured via the particular service's
```
Comment (ollietreend) [inline google-app-scripts/README.md]: > nomrally have to be accesses and configured

2 typos: 'normally' and 'accessed'

File: google-app-scripts/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Google App Scripts for test support
```
Comment (ollietreend) [inline google-app-scripts/README.md]: These are officially called Google Apps Scripts (note the plural 'Apps'), although we've been tending to call them 'App Scripts'.

This doc (and maybe even the directory name) should probably be updated to use the official name, for the sake of correctness.

File: google-app-scripts/google-forms/writeAnswersToFirestore.gas [hunk 3]
```diff
@@ -0,0 +1,21 @@
+function onSubmit(e) {
+  const form = FormApp.getActiveForm();
+  const title = form.getTitle();
+  const fid = form.getId();
+
+  const itemResponses = e.response.getItemResponses();
+
+  const responses = { title: title, fid: fid, responses: {}};
```
Comment (ollietreend) [inline google-app-scripts/google-forms/writeAnswersToFirestore.gas]: Inconsistent spacing when defining this object – there's a space after the opening `{` curly brace, but not before the closing `}`.

File: google-app-scripts/README.md (hunk 1 above)
Comment (tatyree) [reply google-app-scripts/README.md]: No objections at all to you picking me up on typos, misspellings and strange brace spacing.  Fixing.

Comment (tatyree) [reply google-app-scripts/README.md]: You missed 'diffcult' :P.  Fixed that too and have turn on vim's spellchecker (finally).

**Pull request merged**